x-pack/legacy/plugins/snapshot_restore/public/app/services/validation/validate_policy.ts

```diff
@@ -15,6 +15,12 @@ const isStringEmpty = (str: string | null): boolean => {
   return str ? !Boolean(str.trim()) : true;
 };
 
+const isSnapshotNameLowerCase = (str: string): boolean => {
+  const strExcludeDate =
+    str.substring(0, str.search('{')) + str.substring(str.search('}}') + 2, str.length);
+  return strExcludeDate !== strExcludeDate.toLowerCase() ? true : false;
```

Comment:
Can be simplified to:

```suggestion
  return strExcludeDate !== strExcludeDate.toLowerCase();
```

Reply:
Additionally, given the boolean check this function name (`isSnapshotNameLowerCase`) is misleading. Either the name should be `isSnapshotNameNotLowerCase`) or we should change `!==` to `===` and update the `if` logic on L70.

Reply:
> Additionally, given the boolean check this function name (`isSnapshotNameLowerCase`) is misleading. Either the name should be `isSnapshotNameNotLowerCase` or we should change `!==` to `===` and update the `if` logic on L70.

Thanks Jean for the advice. I've changed the name to `isSnapshotNameNotLowerCase` and simplified the return strExcludeDate condition